quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/CLITestExtension.java

```diff
@@ -100,6 +97,24 @@ public void beforeAll(ExtensionContext context) throws Exception {
         super.beforeAll(context);
     }
 
+    @Override
+    public void afterAll(ExtensionContext context) throws Exception {
+        if (dist != null) {
+            //make sure sysprop raw gets unset after tests in class annotated with @DistributionTest finished in raw mode.
+            if (System.getProperty(DIST_TYPE_SYSPROP) != null && System.getProperty(DIST_TYPE_SYSPROP).equals("raw")) {
+                System.setProperty(DIST_TYPE_SYSPROP,"");
```

Comment:
Now we're setting the sysprop in Maven, so it's always set to "raw" if not explicitly set otherwise. As long as we don't need the distinction between whitebox @CLITest and @DistributionTest raw, this suffices and the above code wasn't necessary anymore and thus removed.